packages/sak-app/src/app/paths.ts

```diff
@@ -4,6 +4,7 @@ import { buildPath, formatQueryString, parseQueryString } from '@navikt/ft-utils
 import { skjermlenkeCodes } from '@fpsak-frontend/konstanter';
 
 export const AVDELINGSLEDER_PATH = 'avdelingsleder';
+export const JOURNALFØRING_PATH = 'journalforing';
```

Comment:
Kor blir denne brukt? Bør ein laga path som blir brukt i Home.tsx her, sånn som for dei andre?